.make/test.mk

```diff
@@ -191,6 +191,14 @@ test-remote: prebuild-check $(SOURCES)
 test-migration: prebuild-check
 	AUTH_RESOURCE_DATABASE=1 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test -vet off $(GO_TEST_VERBOSITY_FLAG) github.com/fabric8-services/fabric8-auth/migration
 
+.PHONY: test-e2e
+## Runs the e2e tests
+## Make sure you ran "make integration-test-env-prepare" before you run this target.
+test-e2e: prebuild-check migrate-database $(SOURCES)
+	$(call log-info,"Running test: $@")
+	$(eval TEST_PACKAGES:=$(shell go list ./... | grep integration))
```

Comment:
So, if you do include `test` package here then IMO it's worth keeping the tests into `test/integration` package.

Reply:
only `integration` package is considered.  It's `grep integration`.
@alexeykazakov If you want, I can move files back to `/test/integration` and here only include `/test/integration` package ??

Reply:
Yes. I just don't like packages which contains `only` test related stuff and which don't have `test` in the name :)
Also add this package to `ALL_PKGS_EXCLUDE_PATTERN` so it's not run during integration tests.

Reply:
You actually don't have to include it into `ALL_PKGS_EXCLUDE_PATTERN` if you move it to `/test/integration` because `test` is already excluded in `ALL_PKGS_EXCLUDE_PATTERN`.

Reply:
done [79ba6c1](https://github.com/fabric8-services/fabric8-auth/pull/820/commits/79ba6c12125723c61a371c5bbd0f18bb7ce222a1)